src/openseadragon.js

```diff
@@ -195,8 +195,9 @@
   *
   * @property {String} [compositeOperation=null]
   *     Valid values are 'source-over', 'source-atop', 'source-in', 'source-out',
-  *     'destination-over', 'destination-atop', 'destination-in',
-  *     'destination-out', 'lighter', 'copy' or 'xor'
+  *     'destination-over', 'destination-atop', 'destination-in', 'destination-out',
+  *     'lighter', 'destination', 'copy', 'xor', etc.
```

Comment:
You've added `destination` here, but that's not one of the possible modes... perhaps you meant `difference`?

Reply:
Good catch, Ian!  Yes, that's what I meant.